installer/conf/omsagent.conf

```diff
@@ -6,7 +6,7 @@
 <source>
   type exec
   tag heartbeat.output
-  command /opt/microsoft/omsagent/bin/omsadmin.sh -b > /dev/null
+  command /opt/microsoft/omsagent/bin/omsadmin.sh > /dev/null
   format tsv
   keys severity,message
   run_interval 20m
```

Comment:
We can't without drastically changing the DSC python scripts: https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/master/Providers/Scripts/2.4x-2.5x/Scripts/nxOMSPerfCounter.py#L186
This <source> section does effectively nothing, but removing the -b flag makes that clearer.